docs/en/start/trace/prometheus.md

```diff
@@ -0,0 +1,52 @@
+# Prometheus metrics 
+
+## Run prometheus
+
+window users need to change the layotto in prometheus.yml to 'docker.for.windows.localhost:34903'
+
+![](../../../img/trace/layotto.png)
+
+```shell
+cd ${project_path}/demo/prometheus
+
+docker-compose -f prometheus-docker-compose.yaml up -d
+```
+
+## Run layotto
+
+A layotto server can be started as follows.
+
+```shell @if.not.exist layotto
+cd ${project_path}/cmd/layotto
+# build it
+go build -o layotto
+```
```

Comment:
I suggest:
~~~markdown
```shell 
cd ${project_path}/cmd/layotto
```

Build it:
```shell @if.not.exist layotto
go build -o layotto
```
~~~

So that we can reuse the binary file `${project_path}/cmd/layotto/layotto`